jkind/launch/JKind Main.launch

```diff
@@ -6,7 +6,13 @@
 <listAttribute key="org.eclipse.debug.core.MAPPED_RESOURCE_TYPES">
 <listEntry value="1"/>
 </listAttribute>
+<mapAttribute key="org.eclipse.debug.core.environmentVariables">
+<mapEntry key="YICES2_HOME" value="F:\apps\yices-2.6.0"/>
+<mapEntry key="Z3_HOME" value="F:\apps\z3-4.8.4"/>
+</mapAttribute>
 <stringAttribute key="org.eclipse.jdt.launching.MAIN_TYPE" value="jkind.Main"/>
+<stringAttribute key="org.eclipse.jdt.launching.PROGRAM_ARGUMENTS" value="-jkind -solver yices2 -ivc -scratch gauss.lus"/>
 <stringAttribute key="org.eclipse.jdt.launching.PROJECT_ATTR" value="jkind"/>
+<stringAttribute key="org.eclipse.jdt.launching.WORKING_DIRECTORY" value="F:\scratch"/>
 <stringAttribute key="yk-options" value="&#13;&#10;additional-options2=onexit\=snapshot&#13;&#10;"/>
 </launchConfiguration>
```

Comment:
Remove changes. I recommend having a separate launch config with your customizations that you keep out of git.